server/odc-service/src/main/java/com/oceanbase/odc/service/connection/database/DatabaseService.java

```diff
@@ -574,6 +605,25 @@ public Set<String> filterUnAuthorizedDatabaseNames(Set<String> databaseNames, @N
                 .collect(Collectors.toSet());
     }
 
+    @SkipAuthorize("internal authorized")
+    public Page<DataBaseUser> listUsers(Long dataSourceId) {
```

Comment:
if listAll, List may better than Page

Reply:
this method definition stay the same style like `/api/v2/database/databases` or other previous api